.github/workflows/tests.yml

```diff
@@ -457,9 +461,14 @@ jobs:
           python-version: "3.9"
           environment-file: ci/environment-dashboard.yml
 
+      - name: Download software environment assets
+        uses: actions/download-artifact@v3
+        with:
+          name: software-environment-py3.9
```

Comment:
Where are you using this? Hopefully these assets will be going away soon

Reply:
In the following change (line 471)